src/algo/depth-to-rgb-calibration/optimizer.h

```diff
@@ -63,6 +63,13 @@ namespace depth_to_rgb_calibration {
         double move_thresh_pix_val = 20;
         double move_threshold_pix_num = 62.2080;
 
+        //smearing
+        double max_sub_mm_z = 4;
+        double constant_weights = 1000;
+        double k_depth[3][3] = { {731.27344,0,529.27344 },{0,731.97656,402.32031},{0,0,1} };
```

Comment:
These seem like low precision... they're a copy-paste from matlab, or manually entered?

What's this k_depth of, anyway?

Reply:
Please explain that this is the output of a constant expression run thru a function in algo.
Check if k_depth can be removed.